src/components/Dashboard/TodoList.vue

```diff
@@ -1349,11 +1451,12 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, E
         this.setCurrentFilingStatus(FilingStatus.NEW)
         this.$router.push({ name: Routes.ANNUAL_REPORT, params: { filingId: '0' } }) // 0 means "new AR"
         break
-      case FilingTypes.CONVERSION:
+      case FilingTypes.CONVERSION: { // Curly braces to fix "Unexpected lexical declaration in case block"
```

Comment:
ESLint was complaining about "Unexpected lexical declaration in case block".
![todolist error](https://github.com/bcgov/business-filings-ui/assets/122301442/8d553931-4d7d-4c36-8859-34fb55ff86a3)

Adding curly braces fixes it. Reference: https://eslint.org/docs/latest/rules/no-case-declarations